src/mpid/ch4/include/mpidpre.h

```diff
@@ -481,8 +481,8 @@ typedef enum {
     MPIDI_WINATTR_ACCU_NO_SHM = 4,      /* shortcut of disable_shm_accumulate in MPIDIG_win_info_args_t. */
     MPIDI_WINATTR_ACCU_SAME_OP_NO_OP = 8,
     MPIDI_WINATTR_NM_REACHABLE = 16,    /* whether a netmod may reach the window. Set by netmod at win init.
-                                         * It only indicates whether the win type is supported. Per-target check
-                                         * may be required separately. */
+                                         * Each netmod decides the definition of "reachable" at win_init based on
+                                         * its internal optimization. */
```

Comment:
Saying "the definition depends on ..." is nearly the same as saying nothing. I would rather just remove the comment altogether, then we'll always read the actual code to figure out the meaning -- that is what your new comment says anyway, right?

The current change just removed its usage from ucx netmod. I don't think you need to modify the comment just because some netmod doesn't use this flag. On the other hand, I think it is a good idea to say that this flag is only used by netmod internally -- but that may be already implied by the `NM` namespace.

Reply:
Please see response above. UCX still uses this attribute.

Reply:
I feel that adding this comment does not hurt. Otherwise, people may guess its meaning. At least now someone should check the nm definition rather than guessing